winconf.pri

```diff
@@ -1,68 +1,79 @@
-# Adapt these paths on Windows
-
-# Point this to the boost include folder
-INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_51_0)
-# Point this to the libtorrent include folder
-INCLUDEPATH += $$quote(C:/qBittorrent/RC_0_16/include)
-# Point this to the zlib include folder
-INCLUDEPATH += $$quote(C:/qBittorrent/Zlib/include)
-# Point this to the openssl include folder
-INCLUDEPATH += $$quote(C:/qBittorrent/openssl/include)
-
-# Point this to the boost lib folder
-LIBS += $$quote(-LC:/qBittorrent/boost_1_51_0/stage/lib)
-# Point this to the libtorrent lib folder
-LIBS += $$quote(-LC:/qBittorrent/RC_0_16/bin/path-according-to-the-build-options-chosen)
-# Point this to the zlib lib folder
-LIBS += $$quote(-LC:/qBittorrent/Zlib/lib)
-# Point this to the openssl lib folder
-LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
-
-# BOOST DEFINES
```

Comment:
I hope this comment (and the ones below) could stay (here and in conf.pri.*).

Reply:
Isn't it obvious that the definition like BOOST_* belongs to Boost?

Reply:
yes, I think either there is a comment for each lib or a empty line separating each lib (in conf.pri.*).

Reply:
Ok.